### ISSUE TITLE
Package up for Debian 

### ISSUE THREAD

**Issue by WSLUser** [dankamongmen/libblossom#2]
Despite the changelog shown in the repo, there is no libblossom package available at all on Debian or anywhere else for that matter. Create a binary that's published as a .deb.

**Comment by dankamongmen** [comment]
yep, it would need go through the NEW queue and get approved by FTP team. for that matter, i'm just a debian maintainer at the moment, not a full debian developer, so i'd need to get a sponsor. that should all be done, though i'm not sure i'm ready to do so just yet. this is old code and probably needs a good looking-over.

**Comment by WSLUser** [comment]
Yes, that's why I filed the building issue first. I tried fixing some parts myself but still have something off that's preventing me from getting the Makefile to show up. I wanted to test out Raptorial but you made this a dependency to build first before that.

**Comment by WSLUser** [comment]
Also, it would be nice if you made it available to other Debian distributions like Devuan and Ubuntu. Anything that uses `apt`. Hopefully eventually you can make this available on rpm based distros too.
